cli/genenotebook.js

```diff
@@ -356,35 +356,68 @@ addTranscriptome
         sampleName,
         replicaGroup,
         description,
-      }
+      },
     );
   })
   .exitOverride(customExitOverride(addTranscriptome));
 
-// add interproscan
+// Add interproscan file
 const addInterproscan = add.command('interproscan');
 
 addInterproscan
   .description('Add InterProScan results to a running GeneNoteBook server')
-  .usage('[options] <InterProScan gff3 output file>')
+  .usage('[options] <InterProScan gff3 or tsv output file>')
   .arguments('<file>')
-  .option('-u, --username <username>', 'GeneNoteBook admin username')
-  .option('-p, --password <password>', 'GeneNoteBook admin password')
+  .requiredOption(
+    '-u, --username <adminUsername>',
+    'GeneNoteBook admin username',
+  )
+  .requiredOption(
+    '-p, --password <adminPassword>',
+    'GeneNoteBook admin password',
+  )
   .option(
     '--port [port]',
-    'Port on which GeneNoteBook is running. Default: 3000'
+    'Port on which GeneNoteBook is running. Default: 3000',
   )
-  .action((file, { username, password, port = 3000 }) => {
+  .option(
+    '--parser [parser]',
+    `Choose a parser for an InterProScan output file. Can parse .gff3 and .tsv
+    extensions.`,
```

Comment:
```suggestion
    '--format [format]',
    'Choose the format of the InterProScan output file. Can be "gff3" or "tsv".',
```
"format" is much more clear I think.
Also replaces strange-quotes with normal ones, I don't know if it was intended?